### PR TITLE
Add `update-password` command to `tapi`

### PR DESCRIPTION
**Pull request by pazaan**
@tjotala Adding `update-password` functionality to the `tapi` tool (replacing the `tidepool-pwreset.js` tool).
The corresponding `develop` PR is [available here](https://github.com/tidepool-org/platform/pull/447)

### CODE REVIEW

File: tools/tapi/cmd/user.go [hunk 1]
```diff
@@ -98,6 +98,22 @@ func UserCommands() cli.Commands {
...
+					Name:  "update-password",
+					Usage: "update a user's password",
+					Flags: CommandFlags(
+						cli.StringFlag{
+							Name:  UserIDFlag,
+							Usage: "`USERID` of the user to update",
+						},
+						cli.StringFlag{
+							Name:  PasswordFlag,
+							Usage: "`PASSWORD` of the user to update (required if authenticated as the user being deleted)",
```
Comment (tjotala) [inline tools/tapi/cmd/user.go]: Misleading description? This user is not being deleted, right?

File: tools/tapi/cmd/user.go [hunk 2]
```diff
@@ -195,3 +211,27 @@ func userDelete(c *cli.Context) error {
 
 	return reportMessage(c, "User deleted.")
 }
+
+func userUpdatePassword(c *cli.Context) error {
+	userID := c.String(UserIDFlag)
+	password := c.String(PasswordFlag)
```
Comment (tjotala) [inline tools/tapi/cmd/user.go]: I'm confused. This prompts for the user's current password, but does not appear to use it for anything?

Comment (pazaan) [reply tools/tapi/cmd/user.go]: Good catch! 🤦‍♂

**Comment by pazaan** [comment]
@tjotala Good questions! Updated based on your comments.

**Pull request merged**